tests/system/action/test_internal_actions.py

```diff
@@ -113,37 +118,35 @@ def tearDown(self) -> None:
         del os.environ["INTERNAL_AUTH_PASSWORD_FILE"]
         self.secret_file.close()
 
-    @patch("openslides_backend.shared.env.Environment.is_dev_mode")
-    def test_internal_try_access_backend_internal_action_wrong_pw(
-        self, is_dev_mode: MagicMock
-    ) -> None:
-        is_dev_mode.return_value = False
+    @disable_dev_mode
+    def test_internal_try_access_backend_internal_action_wrong_pw(self) -> None:
         response = self.internal_request(
             "user.create", {"username": "my username"}, "wrong pw"
         )
         self.assert_status_code(response, 401)
         self.assert_model_not_exists("user/2")
 
-    @patch("openslides_backend.shared.env.Environment.is_dev_mode")
-    def test_internal_try_access_backend_common_action(
-        self, is_dev_mode: MagicMock
-    ) -> None:
-        is_dev_mode.return_value = False
+    @disable_dev_mode
+    def test_internal_try_access_backend_common_action(self) -> None:
         response = self.internal_request(
             "user.create", {"username": "my username"}, self.internal_auth_password
         )
         self.assert_status_code(response, 200)
         self.assert_model_exists("user/2")
 
-    @patch("openslides_backend.shared.env.Environment.is_dev_mode")
-    def test_internal_try_access_backend_internal_action(
-        self, is_dev_mode: MagicMock
-    ) -> None:
-        is_dev_mode.return_value = False
+    @disable_dev_mode
+    def test_internal_try_access_backend_internal_action(self) -> None:
         response = self.internal_request(
             "option.create",
             {"meeting_id": 1, "text": "test"},
             self.internal_auth_password,
         )
         self.assert_status_code(response, 400)
         self.assert_model_not_exists("option/1")
+
+    @disable_dev_mode
+    def test_internal_try_access_via_public_route(self) -> None:
+        self.datastore.truncate_db()
+        response = self.request("organization.initial_import", {"data": {}})
+        self.assert_status_code(response, 400)
+        self.assert_model_not_exists("organization/1")
```

Comment:
Show success with internal rout in a new test: `test_internal_try_access_stack_internal_via_internal_route`

Reply:
That test already exists: https://github.com/OpenSlides/openslides-backend/blob/1ed1197d524f5c6f03db7fb7e6ca7ab14ec6f8cb/tests/system/action/test_internal_actions.py#L68

Reply:
I wanted to document in the tests, that an action of type STACK_INTERNAL works on internal route, but not on public route. 
Why do all the ordinary actions should work without user permission, when they were called on the internal route?

Reply:
But exactly that is asserted by the linked test: `organization.initial_import` is a stack internal action and can be accessed via the internal route.

The normal actions must work without user permissions because there is no user whose permissions we can check - the `user_id` comes from the auth service and is based on the provided auth data. The internal route is not called with any auth data, hence we cannot check any permissions.

Reply:
> Proposal: Rename the tests to something with `stack_internal` and internal_route in it's name and/or put it besides the other test concerning the stack_internal-actions. There is to ,much `internal`!

In these tests, `internal == stack_internal` if not said otherwise, since these are supposed to test the stack internal route and actions. I think the current naming is fine.

> Why do we want this? If there are special cases like the SSO-Login with creating/updating users we have specialized actions. To allow any service inside the stack to use all actions without permission-check and user I think this is not a good idea

The manage service may execute arbitrary actions, e.g. to manage users. Also, this is not a new feature introduced with this PR, but it's existed for a rather long time - I don't think we should be having this discussion here.